torchvision/models/mnasnet.py

```diff
@@ -0,0 +1,166 @@
+import math
+
+import torch
+import torch.nn as nn
+
+__all__ = ['MNASNet', 'mnasnet0_5', 'mnasnet0_75', 'mnasnet1_0', 'mnasnet1_3']
+
+# Paper suggests 0.9997 momentum, for TensorFlow. Equivalent PyTorch momentum is
+# 1.0 - tensorflow.
+_BN_MOMENTUM = 1 - 0.9997
+
+
+class _InvertedResidual(nn.Module):
```

Comment:
Is this significantly different from https://github.com/pytorch/vision/blob/74b6a750c7b748d03a3c80f29ebf17ca98dcab8b/torchvision/models/mobilenet.py#L14-L40 ?

Reply:
Forget about this comment for now, let's leave each file in `models` to be self-contained

Reply:
Removed the comment, although yes, I do implement my own MobileNet V2 with the IR block you see in my code. :-)